src/components/table/tableDirective.ts

```diff
@@ -104,6 +101,7 @@ class TableController {
 export interface ITableAttributes extends angular.IAttributes {
   uifRowSelectMode?: string;
   uifTableType?: string;
+  uifSelectedItems?: any;
```

Comment:
Please add documentation in the JSDOC for this interface to document this property.